A6-client-retries.md

```diff
@@ -96,6 +100,10 @@ Each of these configuration options is detailed in its own section below.
 
 gRPC's call deadlines apply to the entire chain of original RPC and retry attempts. The time limit begins when the original RPC is issued to the gRPC client library. For example, if the specified deadline is 500ms and after 500ms only one retry has been attempted, the call is cancelled even if `maxRetryAttempts` is set to two or greater.
 
+![State Diagram](A6_graphics/too_many_attempts.png)
```

Comment:
This diagram is for maxRetryAttempts=2, right? Maybe add this to the image directly or here in the .md file, as our other examples have maxRetryAttempts=3.

Reply:
I changed my diagram to 3 attempts